client/layout/sidebar-v2/navigator/navigator-menu-item.tsx

```diff
@@ -24,6 +24,7 @@ interface Props {
 	isSelected?: boolean;
 	openInSameTab?: boolean;
 	extraContent?: JSX.Element;
+	sidebarIcon?: JSX.Element;
```

Comment:
No need to introduce another Icon prop here. This creates confusion with multiple props that do the same thing. The `icon` prop should be capable of handling regular images.